src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.targets

```diff
@@ -1,381 +1,3 @@
-﻿<Project>
```

Comment:
I plan on moving CodeGeneration.targets in to this file. Kept them separate to make the diff easier to read.